docker/entrypoint.sh

```diff
@@ -27,6 +27,8 @@ case "$1" in
             exec cargo "test" "--locked" "--release" "--features" "enable-sentry,test_sync_to_mandatory_checkpoint_${NETWORK,,}" "--manifest-path" "zebrad/Cargo.toml" "sync_to_mandatory_checkpoint_${NETWORK,,}"
         elif [[ "$TEST_CHECKPOINT_SYNC" -eq "1" ]]; then
             exec cargo "test" "--locked" "--release" "--features" "enable-sentry,test_sync_past_mandatory_checkpoint_${NETWORK,,}" "--manifest-path" "zebrad/Cargo.toml" "sync_past_mandatory_checkpoint_${NETWORK,,}"
+        elif [[ "$TEST_LWD_RPC_CALL" -eq "1" ]]; then
+            exec cargo "test" "--locked" "--release" "--features" "enable-sentry" "--test" "acceptance" "--" "--nocapture" "--ignored" "fully_synced_rpc_test"
```

Comment:
It occurs to me that we pipe in the test name/id in multiple places but that does not determine what test is executed inside the container image, only `$TEST_LWD_RPC_CALL"` does, and that comes from the test variables, and the test name is hardcoded. We should consolidate so that skew in the naming and test execution is less likely to happen and less likely to break, but that doesn't have to be done in this PR

Reply:
Related discussion: https://github.com/ZcashFoundation/zebra/discussions/3683

Reply:
This isn't on the critical path to getting the send transaction test working in CI. So we can follow up by finishing that discussion, then creating low priority tickets.

Reply:
👍👍👍